docs/project/2024/Index.md

```diff
@@ -0,0 +1,104 @@
+# Tic-Tac_Toe
+This project aims to emulate the popular game Tic-Tac_Toe on a Rapspberry Pi.
+
+**Author**: Teodor-Matei FRÂNCU \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-Angery-Stoman)
+
+
+
+## Description
+
+
+In this project I will try to recreate simplified version of a game console of the 1990's. The controller for playing the game will be composed of 5 buttons, 4 of which for navigating the game itself (up down left right) and the last for inputting your move. The game will be displayed using a small LCD display that will run using Embedded-graphics, thereby giving the user a simple yet effective interface for playing the game. To emulate the feel of retro games, a buzzer (or more, if need be) will be used to play a simple 8-bit styled theme and specific sounds for inputting moves and losing or winning the game.
+The game will be played against a bot that is coded using some simple checks and ifs.
+
+
+## Motivation
+
+I've always wanted to delve into game making, and while this may not be much, it's a small step into the world of game making. Tic Tac Toe feels simple enough for someone of my hardware knowledge level to recreate it. Choosing a game as my project also adds a level of difficulty to the journey of creating it, as I have only coded games using html, c++/c# and javascript, a.k.a. only software. This will be the first time any game that I have made has a hardware component to it.
+And as an added bonus I'm quite passionate about video games and I spend most of my free time playing them, so this project is right up my alley.
+
+## Architecture 
+
+![architecture](<img width="870" alt="architecture" src="https://github.com/UPB-FILS-MA/upb-fils-ma.github.io/assets/163386844/781248c2-bc65-4d6b-a6b0-360f5891a9d4">)
+
+### Main Components
+
+Here is the list of all of the main components and a short description of their utility:
+
+-Controller
```

Comment:
Is this displayed correctly?

Reply:
how can I check if it is displayed correctly?